runtime/runtime/src/congestion_control.rs

```diff
@@ -792,8 +793,9 @@ impl<'a> DelayedReceiptQueueWrapper<'a> {
         let receiver_id = receipt.get_receipt().receiver_id();
         let receipt_shard_id = self
             .epoch_info_provider
-            .account_id_to_shard_id(receiver_id, &self.epoch_id)
-            .expect("account_id_to_shard_id should never fail");
+            .shard_layout(&self.epoch_id)
+            .unwrap()
```

Comment:
This unwrap bothers me a little but it's not related to your PR. Please ignore unless you have any good ideas.